test/ui-test/testSuites/suite_messaging/tst_chatsNavigation/test.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Status Desktop Chat Navigation
+
+    As a user I want to join seethe application reflect correctly
+    when I navigate trough chats list.
+
+    The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
+
+	[Cleanup] Also each scenario starts with:
+	** when the user opens the chat section
+
+    Scenario: The user joins a room and marks it as read
+        When the user joins chat room "test"
+        And the user marks the channel "test" as read
+        # TODO find a way to validate that it worked
+
+    Scenario: The user sees chats sorted by most recent activity
```

Comment:
Would prefer it if these had a Given step and if they were split into 2 scenarios, but they're probably specific enough that it's not an issue.

Suggesting a minor revision:

```
When the user joins the chat named "first-chat"
And the user joins the chat named "second-chat"
And the user joins the chat named "third-chat"
Then the chat navigation is sorted by most recent first
| third-chat      |
| second-chat  |
| first-chat        |

When the user switches to "second-chat" chat
And the user sends a random chat message
Then the random chat message is displayed
Then the chat navigation is sorted by most recent first
| second-chat |
| third-chat      |
| first-chat       |
```

If it's useful for other scenarios, the step could also be configured to accept all the required chats in one go:
```
When the user joins chats
| first-chat       |
| second-chat  |
| third-chat       |
Then the user chats are sorted by most recent first
| third-chat      |
| second-chat  |
| first-chat        |